rules.txt

```diff
@@ -128,6 +128,7 @@
 /commercial-activity-license              301 /services/permits-violations-licenses/get-a-license/business-licenses-permits-and-approvals/get-a-commercial-activity-license/
 /complete-streets/chestnut-walnut-traffic-safety 301 /programs/complete-streets/projects/chestnut-walnut-repaving-and-complete-street-safety-project/ 
 /complete-streets/west-chestnut-walnut    301 /programs/complete-streets/projects/chestnut-walnut-repaving-and-complete-street-safety-project/
+/complete-streets/penrose-traffic-calming 301 /2021-08-12-penrose-roundabout-looks-to-improve-safety-at-busy-intersection/
```

Comment:
Sorry this is a little nit-picky, but can you double check there are two spaces after the protocol to align with the rest of the destinations?

Reply:
Yep it's been fixed.